zathura/database-sqlite.c

```diff
@@ -178,7 +179,7 @@ sqlite_db_init(ZathuraSQLDatabase* db, const char* path)
   static const char SQL_FILEINFO_ALTER2[] =
     "ALTER TABLE fileinfo ADD COLUMN first_page_column INTEGER;";
```

Comment:
This should be changed to `STRING` so that we can skip the extra step at the end.